src/applications/check-in/components/pages/AppointmentDetails/index.jsx

```diff
@@ -94,6 +96,37 @@ const AppointmentDetails = props => {
       </p>
     );
   }
+  if (isCvtAppointment) {
+    preCheckInSubTitle = (
+      <p data-testid="cvt-appointment-subtitle" className="vads-u-margin--0">
+        {t('go-to-facility-for-this-video-appointment', {
+          facility: appointment.facility,
+        })}
+      </p>
+    );
+  }
+  if (isVvcAppointment) {
+    preCheckInSubTitle = (
+      <p data-testid="vvc-appointment-subtitle" className="vads-u-margin--0">
+        {t('you-can-join-your-appointment-by-using-our-appointments-tool')}
+      </p>
+    );
+  }
+
+  const appointmentTitle = () => {
+    if (isPhoneAppointment) {
```

Comment:
Maybe not this exactly, but something like this might help here:

```
const appointmentTitle = () => {
    switch (appointment?.kind) {
      case 'phone':
        return `${t('phone')} ${t('appointment')}`;
      case 'cvt':
        return t('video-appointment-at-facility', { facility: appointment.facility });
      case 'vvc':
        return t('video-appointment--title');
      default:
        return t('in-person-appointment');
    }
  }
  ```

Reply:
Good idea, I changed it.